README.md

```diff
@@ -14,12 +14,14 @@ Authors (in alphabetical order): Katharina Rasch, Noa Tamir
 
 ## Goal
 
-By the end of this lab you will have completed one data science project based on open data, simulating the work of a freelance data science team.
+By the end of this lab you will have completed one data science project based on open data, simulating the work of a freelance data science team. 
 
 You will gain familiarity with processes and practice the required communication skills at the core of a data scientist’s job. 
 
 You will get experience with weekly planning, receiving and following guidance by professional data science tech leads, team collaboration, communication to stakeholders, documentation for project handover, sharing your ongoing work and working together on not-perfect code and ideas, peer 2 peer code review.
 
+And, importantly, you will end up with a github repository (or alternative material such as blog post, presentations) showcasing your work on a real-world data problem, which you can use in job applications.  
```

Comment:
"... you will end up with a portfolio project (in the form of a presentation, blog post, github repository, or another format you prefer) showcasing your work ...."

Reply:
thanks, fixed